From 7f490a62c10368d9ec01ff1c9778ff93bf842ec8 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Sun, 9 Oct 2016 16:02:40 +0800
Subject: [PATCH 421/666] usb: dwc3: core: add the missing unified device
 interface replacement

Reference the commit 3d128919b7e5 (usb: dwc3: core: convert to unified device
property interface), use the unified device interface to replace the
"of_property*" APIs, here add the missed ones and to fix below build error:

drivers/usb/dwc3/core.c:989:6: error: 'node' undeclared (first use in this function)
|   if (node && of_dma_is_coherent(node)) {
|       ^~~~

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/usb/dwc3/core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/dwc3/core.c b/drivers/usb/dwc3/core.c
index 7994fbf..2d091ba 100644
--- a/drivers/usb/dwc3/core.c
+++ b/drivers/usb/dwc3/core.c
@@ -85,7 +85,7 @@ static int dwc3_get_dr_mode(struct dwc3 *dwc)
 		break;
 	default:
 	/* Enable Snooping */
-	if (node && of_dma_is_coherent(node)) {
+	if (device_property_read_bool(dev, "dma-coherent")) {
 		dwc3_writel(dwc->regs, DWC3_GSBUSCFG0,
 		dwc3_readl(dwc->regs, DWC3_GSBUSCFG0) | 0x22220000);
 		dev_dbg(dev, "enabled snooping for usb\n");
-- 
2.7.4

